VLogProcessor/src/edu/umich/robustopt/util/SchemaUtils.java

```diff
@@ -44,6 +46,10 @@
 	}
 
 	public static SchemaDescriptor GetSchemaMapFromDefaultSources(String dbName, String VendorSpecificDatabaseLoginName) throws Exception {
+		if (VendorSpecificDatabaseLoginName.equalsIgnoreCase("vertica"))
+			VendorSpecificDatabaseLoginName = VerticaDatabaseLoginConfiguration.class.getSimpleName();
+		else if (VendorSpecificDatabaseLoginName.equalsIgnoreCase("microsoft"))
+			VendorSpecificDatabaseLoginName = MicrosoftDatabaseLoginConfiguration.class.getSimpleName();
```

Comment:
Sometimes this function is called with parameter DBVendor in the place of VendorSpecificDatabaseLoginName.
```Java
	public LogFileBasedEuclideanDistanceWorkloadGeneratorUnionOfAllClauses(String dbName, String DBVendor, List<String> allPossibleSqlQueries, Set<UnionOption> whichClauses) throws Exception{
		this(SchemaUtils.GetSchemaMapFromDefaultSources(dbName, DBVendor).getSchemas(), null, allPossibleSqlQueries, whichClauses);
```